### PR TITLE
Features/ui fixes

### PR DESCRIPTION
**Pull request by AkuaQ**
Added core data and made UI changes

### CODE REVIEW

**Comment by AkuaQ** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- FirstApp/Recipes/View/Cells/CellTableViewCell.swift  1
- FirstApp/Recipes/View/Cells/RecipeCellTableViewCell.swift  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/AkuaQ/MyFirstIoSApp/pullRequest?prid=5429213&bid=17803409)

File: FirstApp/Favourites/ProfileRepo.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        let entity =
+          NSEntityDescription.entity(forEntityName: "Camera",
+                                     in: managedContext)!
+        let photo = NSManagedObject(entity: entity,
+                                     insertInto: managedContext)
+        photo.setValue(name, forKeyPath: "photo")
+        do {
+          try managedContext.save()
+        } catch let error as NSError {
+          print("Could not save. \(error), \(error.userInfo)")
```
Comment (JancoDVTDev) [inline FirstApp/Favourites/ProfileRepo.swift:29]: Rather display the error with an alert, instead of printing

**Review by JancoDVTDev — APPROVED** [review]
Everting else looks good

File: FirstApp/Favourites/ProfileRepo.swift (hunk 1 above)
Comment (AkuaQ) [reply FirstApp/Favourites/ProfileRepo.swift:29]: Interesting idea. Thanks, I'll look into it when I get back to this task again.

Comment (JancoDVTDev) [reply FirstApp/Favourites/ProfileRepo.swift:29]: Cool, if you don't wanna display and it needs to have code in the braces, use return

**Pull request merged**